regression/cbmc/runtime-profiling/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+--
+--
+To test runtime profiling output.
```

Comment:
I don't see anything in this test that actually checks this.

Reply:
> How does this differ to `--timestamp` or piping the output through `ts` or similar?

With the --timestamp option, we get timestamps for every line of output from CBMC, but this may not give the correct duration of a specific stage in CBMC as it may include more than just a call to the function that performs operations for that specific stage. So here we look at the duration of a phase in CBMC, for eg post processing, by tracking the monotonic clock time right before and right after the call to `post_process()`. Similarly, for other phases: symex, ssa conversion and solver.

Reply:
Wasn't sure what to add here. It's not always the case that all stages of CBMC are executed for different tests so the output varies and only the lines specified here are common.

Reply:
I can provide a general test along the lines of ("Runtime Symex: .*" | "Runtime Convert SSA: .*" ...)

Reply:
@natasha-jeppu but this only testing one file with one set of arguments? Surely we'd expect the output here to be somewhat consistent?

Reply:
The only output that was consistent across all tests was "Runtime Solver: .*". Is there a way to access the CI checks done for previous commits? It is evident there.